app/views/layouts/notifications.text.erb

```diff
@@ -0,0 +1,5 @@
+<%= yield %>
+
+<% if !/production/i.match?(GovukAdminTemplate.environment_label) -%>
```

Comment:
"calls to action" would be more correct

Reply:
Do we also have access to `is_production?` here?

Reply:
That's a good point, I didn't think we did, but maybe we do...

Reply:
I don't remember the exact behaviour of actionmailer but it's heavily modelled on controllers/views so I suspect if it's not already available as a helper method we might be able to make it one.